src/codegen/templates/system/system.py.mako

```diff
@@ -372,25 +372,18 @@ ${function_template.script_function(function_object)}
               Specifies the output type for the physical channel
               specified with the **physical_channel** input.
         """
-        channel_names = []
-        channel_types = []
-
         device = _DeviceAlternateConstructor(device_name, self._interpreter)
+        channel_names = device.ao_physical_chans.channel_names
 
-        for ao_physical_chan in device.ao_physical_chans:
-            channel_type = ctypes.c_int()
-            channel_types.append(channel_type)
-            channel_names.append(ao_physical_chan.name)
-
-        states = self._interpreter.get_analog_power_up_states(device_name, channel_names, channel_types)
+        states, channel_types = self._interpreter.get_analog_power_up_states_with_output_type(flatten_channel_string(channel_names), len(channel_names))
```

Comment:
I think you can replace the rest of this with `return self.get_analog_power_up_states_with_output_type(channel_names)`